celery-stubs/app/base.pyi

```diff
@@ -55,7 +55,8 @@ class Celery:
         set_as_current: bool = ...,
         tasks: str | type[TaskRegistry] | None = ...,
         broker: str | None = ...,
-        include: list[str] | None = ...,
+        imports: list[str] | tuple[str, ...] | None = ...,
+        include: list[str] | tuple[str, ...] | None = ...,
```

Comment:
I've decided to extend it with a `tuple[str, ...]` since I got an original type from here https://github.com/celery/celery/blob/1f27d71c5f54da828a34d12fe30c22cedc893272/celery/app/defaults.py#L71

Reply:
Do you think `Sequence[str]` makes sense? I wonder if they're mutating it internally